larpandoracontent/LArObjects/LArCaloHit.h

```diff
@@ -200,16 +248,27 @@ inline pandora::StatusCode LArCaloHitFactory::Read(Parameters &parameters, pando
 {
     // ATTN: To receive this call-back must have already set file reader mc particle factory to this factory
     unsigned int larTPCVolumeId(std::numeric_limits<unsigned int>::max());
+    unsigned int daughterVolumeId(std::numeric_limits<unsigned int>::max());
 
     if (pandora::BINARY == fileReader.GetFileType())
     {
         pandora::BinaryFileReader &binaryFileReader(dynamic_cast<pandora::BinaryFileReader&>(fileReader));
         PANDORA_RETURN_RESULT_IF(pandora::STATUS_CODE_SUCCESS, !=, binaryFileReader.ReadVariable(larTPCVolumeId));
+        if(m_version>1)
```

Comment:
Cosmetics. Indentation of parentheses, and quite a big splash in implementation for operation. Maybe initialise daughterVolumeId to zero, and have a single line `if (m_version > 1) PANDORA...`

Note spaces around operator>

Here's where we start to actually enforce the versioning scheme, of course. Here we check > 1, so that suggests new volume id in version 2 onwards. We should just check what we want to set as a precedent for versioning.